libs/logic-apps-shared/src/designer-client-services/lib/connection.ts

```diff
@@ -38,7 +38,7 @@ export interface CreateConnectionResult {
 
 export interface IConnectionService {
   getConnector(connectorId: string): Promise<Connector>;
-  getConnectorAndSwagger(connectorId: string): Promise<ConnectorWithSwagger>;
+  getSwaggerFromConnector(connectorId: string): Promise<OpenAPIV2.Document>;
   getSwaggerFromUri(uri: string): Promise<OpenAPIV2.Document>;
```

Comment:
You need to mark this as breaking change so other teams can update their connection service

Reply:
Yep I noted that this was a breaking change up in the description